File: 485-max-consecutive-ones/485-max-consecutive-ones.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
         # [1,1,0,1,1,1] , len = 6
            
        right = 0 # 
        count = 0 # 
        result = 0 # 
        
        while right < len(nums):
            if nums[right] == 0 :
                result = max(result, count)
                right = right + 1
                count = 0
            else:
                count += 1
                right += 1
            if right == len(nums):
                result = max(result, count)
            
                
        return result
        

File: 485-max-consecutive-ones/485-max-consecutive-ones.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findMaxConsecutiveOnes = function(nums) {
    let count = 0 
    let result = 0       
    let right = 0
    while(right < nums.length){
        if (nums[right] == 0) {
            result = Math.max(result, count)
            right += 1
            count = 0
        }else{
            count += 1
            right += 1
        }                   
        if (right == nums.length){
            result = Math.max(result, count)
        }
            
                
    }
        
        return result
};